modules/users/server/controllers/users/users.profile.server.controller.js

```diff
@@ -642,6 +644,10 @@ exports.sanitizeProfile = function(profile, authenticatedUser) {
   delete profile.password;
   delete profile.salt;
 
+  // This information is not sensitive, but isn't needed at frontend
+  delete profile.publicReminderCount;
```

Comment:
True, these are getting hard to manage.

Added issue; https://github.com/Trustroots/trustroots/issues/402